packages/rum/src/domain/record/serialization/serializeNode.spec.ts

```diff
@@ -1,43 +1,36 @@
 import { isIE, noop } from '@datadog/browser-core'
 
 import type { RumConfiguration } from '@datadog/browser-rum-core'
-import { STABLE_ATTRIBUTES, DEFAULT_PROGRAMMATIC_ACTION_NAME_ATTRIBUTE } from '@datadog/browser-rum-core'
 import {
   NodePrivacyLevel,
   PRIVACY_ATTR_NAME,
   PRIVACY_ATTR_VALUE_ALLOW,
   PRIVACY_ATTR_VALUE_HIDDEN,
   PRIVACY_ATTR_VALUE_MASK,
   PRIVACY_ATTR_VALUE_MASK_USER_INPUT,
-} from '../../constants'
-import { isAdoptedStyleSheetsSupported } from '../../../../core/test/specHelper'
+} from '../../../constants'
 import {
   HTML,
-  AST_ALLOW,
+  generateLeanSerializedDoc,
   AST_HIDDEN,
   AST_MASK,
   AST_MASK_USER_INPUT,
-  generateLeanSerializedDoc,
-} from '../../../test/htmlAst'
-import type { ElementNode, SerializedNodeWithId, TextNode } from '../../types'
-import { NodeType } from '../../types'
-import type { IsolatedDom } from '../../../../rum-core/test/createIsolatedDom'
-import { createIsolatedDom } from '../../../../rum-core/test/createIsolatedDom'
+  AST_ALLOW,
+} from '../../../../test/htmlAst'
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏it seems that htmlAst is only used in serialization tests, if it is the case what about moving this utility to the serialization package?

Reply:
For test utilities, we are setting them in `/test` folders. Should I create `serialization/utils/test/htmlAst.ts` ?

Reply:
Mostly for shared one and at the same level than `src` though 🤔
For non shared utilities, most of the time they are in the same file as the spec that use them.
I don't know if we have some cases with test utilities that are in a dedicated file but used only by a single file, it could worth to have a look. 
Instead of adding an extra directory, we could have a file suffix to easily identify those files, something like `htmlAst.specHelper.ts` since we already have a couple of `specHelpers.ts`.
wdyt?

Reply:
Sounds good 👍

Reply:
I'll update the code convention